App.tsx

```diff
@@ -1,20 +1,78 @@
-import { StatusBar } from 'expo-status-bar';
-import { StyleSheet, Text, View } from 'react-native';
+import { useState } from "react";
+import { StyleSheet, View, FlatList, ScrollView } from "react-native";
+import TodoItem from "./components/TodoItem";
+import TodoInput from "./components/TodoInput";
+import { Image } from "react-native";
 
 export default function App() {
+  const [todosList, setTodosList] = useState([]);
+
+  function addTodoText(enteredTodoText) {
```

Comment:
Please add typescript annotation to param as it's unknown

Reply:
"string" is fine or pass the "any "